openupgrade_scripts/scripts/website_slides/17.0.2.7/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,82 @@
+---Fields in module 'website_slides'---
+website_slides / res.partner              / slide_channel_ids (many2many) : not stored anymore
+website_slides / res.partner              / slide_channel_ids (many2many) : now a function
+website_slides / res.partner              / slide_channel_ids (many2many) : table is now 'False' ('slide_channel_partner')
+# NOTHING TO DO
+
+website_slides / slide.answer             / _order                        : _order is now 'question_id, sequence, id' ('question_id, sequence')
+website_slides / slide.channel            / activity_user_id (many2one)   : not related anymore
+website_slides / slide.channel            / activity_user_id (many2one)   : now a function
+website_slides / slide.channel            / channel_partner_all_ids (one2many): NEW relation: slide.channel.partner
+website_slides / slide.channel            / karma_gen_slide_vote (integer): DEL
+website_slides / slide.channel            / message_main_attachment_id (many2one): DEL relation: ir.attachment
+website_slides / slide.channel            / partner_ids (many2many)       : not stored anymore
+website_slides / slide.channel            / partner_ids (many2many)       : now a function
+website_slides / slide.channel            / partner_ids (many2many)       : table is now 'False' ('slide_channel_partner')
+website_slides / slide.channel            / prerequisite_channel_ids (many2many): NEW relation: slide.channel
+website_slides / slide.channel            / prerequisite_of_channel_ids (many2many): NEW relation: slide.channel
+website_slides / slide.channel            / visibility (selection)        : selection_keys is now '['connected', 'members', 'public']' ('['members', 'public']')
+# NOTHING TO DO
+
+website_slides / slide.channel.partner    / active (boolean)              : NEW hasdefault: default
+website_slides / slide.channel.partner    / completed (boolean)           : DEL
+website_slides / slide.channel.partner    / last_invitation_date (datetime): NEW
+website_slides / slide.channel.partner    / member_status (selection)     : NEW required, selection_keys: ['completed', 'invited', 'joined', 'ongoing']
```

Comment:
needs to fill using `completion` field:
- 'joined' : enrolled member, not started (`completion` == 0)
- 'ongoing' : enrolled member, started but never finished (0 < `completion` < 100)
- 'completed' : enrolled member who completed the course (`completion` == 100)

'invited' is new feature.